### PR TITLE
blog: how to send and receive emails using Node-RED

### LINKED ISSUE #1940: Blog art : How to Send and Receive Emails using Node-RED
### Related Issue

_No response_

### Description

**Considerations:**

Blog tile : How to Send and Receive Emails using Node-RED

**Inspirations:**
content refference : [How to Send and Receive Emails using Node-RED](https://github.com/FlowFuse/website/pull/1945)

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Related Issue(s)

closes #1940 
closes https://github.com/FlowFuse/customer/issues/186
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2024/04/how-to-send-and-receive-emails-using-node-red.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+5. **TLS Option:**
+   - Checking this option ensures that the email node verifies the validity of the server's SSL/TLS certificate. It's recommended to enable this option to ensure secure communication with the mail server.
+
+6. **Format to SASL:**
+   - SASL (Simple Authentication and Security Layer) XOAuth2 tokens are used for authentication when using the XOAuth authentication method.
+   - If this option is ticked, the email node will automatically format the XOAuth2 token by combining the username and token, encoding it in base64, and passing it to the mail server in the correct format.
+   - If this option is unticked, you must manually format the token before passing it to the email node.
+
+7. **Protocols:**
```
Comment (ZJvandeWeg) [inline src/blog/2024/04/how-to-send-and-receive-emails-using-node-red.md:69]: Merge this one with point 1?

Comment (ZJvandeWeg) [reply src/blog/2024/04/how-to-send-and-receive-emails-using-node-red.md:69]: Point 2 also needs this information

File: src/blog/2024/04/how-to-send-and-receive-emails-using-node-red.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+### Best practices to follow 
+
+- Clear and Concise Messaging: Keep notification messages clear, concise, and action-oriented. Communicate the purpose of the notification and any necessary next steps that recipients should take.
+- Avoid spam trigger words: Refrain from using words commonly associated with spam in your email content, such as "free," "limited time offer," or "urgent."
+- Ensure your email is authenticated: Perform email authentication using protocols like SPF, DKIM, and DMARC to verify the legitimacy of your emails and improve deliverability.
+- Manage Email Frequency: Avoid sending too many email notifications within a short period, as this can trigger spam filters. Instead, maintain a consistent sending frequency and provide valuable content to recipients.
+- Maintain a Clean Email List: Regularly clean your email list by removing invalid or inactive email addresses. High bounce rates and spam complaints can negatively impact your sender's reputation.
+- Monitor Sending Reputation: Monitor your sender reputation using tools like SenderScore or Postmaster Tools. A poor sender reputation can result in email deliverability issues and increased spam filtering.
```
Comment (ZJvandeWeg) [inline src/blog/2024/04/how-to-send-and-receive-emails-using-node-red.md:153]: Can we have less lists/bullet points/ etc? Just text explaining?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/blog/2024/04/how-to-send-and-receive-emails-using-node-red.md (hunk 2 above)
Comment (sumitshinde-84) [reply src/blog/2024/04/how-to-send-and-receive-emails-using-node-red.md:153]: Got it, I'll replace lists and bullet points with descriptive text.

**Comment by ZJvandeWeg** [comment]
@sumitshinde-84 Any reason this is closed?

**Comment by sumitshinde-84** [comment]
@ZJvandeWeg I am creating a new one. I made some mistakes by accident in this PR. I apologize for them

**Pull request closed without merging**